packages/ui-react/src/containers/ShelfList/ShelfList.tsx

```diff
@@ -75,7 +75,7 @@ const ShelfList = ({ rows }: Props) => {
           const posterAspect = parseAspectRatio(playlist.cardImageAspectRatio || playlist.shelfImageAspectRatio);
           const visibleTilesDelta = parseTilesDelta(posterAspect);
 
-          const translatedKey = custom?.[`title-${language}`];
+          const translatedKey = custom?.[`title-${language}`] || (playlist?.[`title-${language}`] as string);
```

Comment:
```suggestion
          const translatedKey = custom?.[`title-${language}`] || (playlist?.[`title-${language}`] as string | undefined);
```

Reply:
TypeScript already infers that the value could be undefined due to optional chaining (?.).

Reply:
You're bypassing that because of the typecast. @langemike